src/Illuminate/Contracts/Foundation/Application.php

```diff
@@ -82,6 +82,13 @@ public function runningInConsole();
      */
     public function runningUnitTests();
 
+    /**
+     * Get the instance of the MaintenanceMode.
+     *
+     * @return \Illuminate\Foundation\MaintenanceMode
+     */
+    public function maintenanceMode();
```

Comment:
Forgot to mention but you'll need to remove this change since this is still a breaking change.

Reply:
I would suggest in that case to leave the PR to 9.x open instead? Introducing the method without updating the interface feels invalid to me.

Reply:
Yeah that's okay 👍